recipes/graylog-illuminate/recipe.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../tools'
+
+class GraylogIlluminate < FPM::Cookery::Recipe
+  description 'Graylog Illuminate'
+
+  name     'graylog-illuminate'
+  version  data.version
+  revision data.revision
+  homepage 'https://graylog.org'
+  arch     'amd64'
+
+  source data.source
+  sha256 data.sha256
+
+  maintainer 'Graylog, Inc. <hello@graylog.org>'
+  vendor     'graylog'
+  license    data.license
+
+  config_files   Dir['graylog_illuminate_core/*'],
+                 Dir['graylog_illuminate_windows/*'],
+                 Dir['graylog_illuminate_okta/*'],
+                 Dir['graylog_illuminate_paloalto/*'],
+                 Dir['graylog_illuminate_o365/*'],
+                 Dir['illuminate_elastic_template/*']
```

Comment:
This doesn't work. It would set the config files in the package to `/path/to/whereever/the/recipe/is/tmp-build/illuminate-1.5../graylog_illuminate_core/`.

We need to use absolute paths here. Like `/etc/graylog/graylog_illuminate_core`. Also, we have to declare each file instead of using a wildcard.

Reply:
That's a bummer. I was really hoping to avoid including specific filenames there. Because any time we add more csv files, we'll also have to update the package before rebuilding it!

Reply:
Okay, I can see that. What we could do is the following. That iterates over the existing files and then sets an explicit path for it:

```suggestion
  config_files [
    'graylog_illuminate_core',
    'graylog_illuminate_windows',
    'graylog_illuminate_okta',
    'graylog_illuminate_paloalto',
    'graylog_illuminate_o365',
    'illuminate_elastic_template'
  ].map {|type|
    Dir["#{type}/*"].map {|f|
      File.join("/etc/graylog/#{type}", File.basename(f))
    }
  }.flatten
```

Reply:
Thanks, that's helpful.